### PR TITLE
Change schema direct accessing to scheme getter.

### PR DESCRIPTION
**Pull request by SerafimArts**
An ORMException with message "ORM is not configured, schema is missing" shoud throws insted of Fatal PHP Error.

### CODE REVIEW

File: src/ORM.php [hunk 1]
```diff
@@ -65,7 +65,7 @@ public function __construct(FactoryInterface $factory, SchemaInterface $schema =
     {
         $this->generator = new CommandGenerator();
         $this->factory = $factory;
-        $this->schema = $schema;
+        $this->schema = $schema ?? new Schema([]);
```
[the remark this answers is not in the document]
Comment (SerafimArts) [reply src/ORM.php]: See first commit https://github.com/cycle/orm/pull/19/commits/b0e951314bf22b1d01d139fb4e213c03b87d132c

I can revert it. At the moment, with an undefined Schema, a null pointer exception is called everywhere.

[the remark this answers is not in the document]
Comment (SerafimArts) [reply src/ORM.php]: P.S. For example:
`->resolveRole` throws PHP Fatal Error
`->make` throws PHP Fatal Error
`->getMapper` throws PHP Fatal Error

etc.

There is 2 options:
1) initialize empty schema (as I did in the second case).
2) or always use a `getSchema` getter.

[the remark this answers is not in the document]
Comment (wolfy-j) [reply src/ORM.php]: Maybe then you can remove the schema exception. I agree on using default empty shema.

[the remark this answers is not in the document]
Comment (SerafimArts) [reply src/ORM.php]: @wolfy-j done. At the same time corrected another place with a possible NPE error

**Comment by wolfy-j** [comment]
Thank you!

**Pull request merged**